packages/deployment/lib/message.js

```diff
@@ -135,6 +135,7 @@ function template(name, workflow, config, outputs) {
     },
     meta: {
       workflow_name: name,
+      workflow_tasks: {},
```

Comment:
When does this get populated? In my execution it's still empty.

Reply:
The only missing commit for these tests is to update all of the `package.json` files with the new message adapter versions. Once those PRs are merged the versions published, these tests will have the updated functionality.